chainstate/tx-verifier/src/transaction_verifier/storage.rs

```diff
@@ -65,6 +67,10 @@ pub trait TransactionVerifierStorageRef: UtxosStorageRead {
         block_id: &Id<GenBlock>,
     ) -> Result<Option<GenBlockIndex>, storage_result::Error>;
 
+    fn get_is_mainchain_tx_index_enabled(
```

Comment:
I think this has to go.

Reply:
Replaced with new `TransactionVerifierConfig` config.